Automattic/wp-calypso
# Sites dashboard v2 - Add/status filter functionality to v2 sites dashboard

Related to # https://github.com/Automattic/dotcom-forge/issues/6711

## Proposed Changes

* Adds filtering functionality to the v2 sites dashboard.
* Note - pagination, sorting, etc. are not yet connected and scoped for follow up PRs.

BEFORE
![Screenshot 2024-04-24 at 2 48 54 PM](https://github.com/Automattic/wp-calypso/assets/28742426/ce3f7694-9a1e-4708-91f2-00d31b7b4e41)


AFTER
![Screenshot 2024-04-24 at 2 49 27 PM](https://github.com/Automattic/wp-calypso/assets/28742426/2e886393-432b-48e7-9bca-9a7503f73fa2)



## Testing Instructions



* Run this calypso branch.
* Visit the sites dashboard */sites with the feature flag ?flags=layout/dotcom-nav-redesign-v2 added to the url.
* Test the filters dropdown in the sites dashboard and select statuses.
* Verify the sites list updates and shows only sites of the selected status.
* Verify the selected status as been added to the URL as a query param.
* Reload the browser window, the sites dashboard should initialize with the status from the URL and the list should show only sites of this status as expect.
* Test alongside search input queries as well.

## Pre-merge Checklist
